docs/tasks/ros-image.md

```diff
@@ -66,6 +76,20 @@ jupyter-repo2docker --no-run --no-clean --user-name jovyan --user-id 1000 --push
 
 ## Add Image to PrimeHub
 
-After the image tagged`infuseai/ros` is pushed, add it to the PrimeHub and start Notebook with the image in `Safe Mode`.
+Add Docker image into PrimeHub images function.
+
+![](assets/task_ros_create_image.png)
+
+Start Notebook with the image in `Safe Mode`.
+
+![](assets/task_safe_mode.png)
+
+After starting the Notebook, please git clone jupyter notebooks of the example.
+
+```bash
+git clone https://github.com/RoboStack/jupyter-ros.git
+```
+
+- Open `notebooks/ROS 3D Grid.ipynb` jupyter notebook and click `circle icon` to open live demo.
```

Comment:
@LiuYuWei  Simon, please annotate the screenshot to highlight where the "circle icon" is. thx